UE/Application/Application.hpp

```diff
@@ -36,10 +36,12 @@ class Application : public IEventsHandler
     void handleAttachReject() override;
     void handleSms(const Sms& sms) override;
     void handleCallRequest(common::PhoneNumber from) override;
+    void handleCallDropped(common::PhoneNumber from) override;
```

Comment:
I would change name of function to `handleReceiveCallDrop`. It would be more consitent, because in diffrent stories you will must send/receive requests, drops. Just handleCallDropped it doesn't point is handle for received drop from BTS or sending call drop from UE. This 

Example:
`handleCallDropped`  -> `handleReceiveCallDrop`  (receiving call drop from BTS)
`handleCallDrop` -> `handleSendCallDrop`             (sending call drop to BTS)